noark-extraction-validator/src/main/java/com/documaster/validator/validation/utils/PDFAValidator.java

```diff
@@ -45,37 +54,16 @@ public static String getFileType(File file) throws IOException {
 
 	public static boolean isValidPdfaFile(File file) throws IOException {
```

Comment:
Again no longer needed.
```suggestion
	public static boolean isValidPdfaFile(File file) {
```

Reply:
Applied w/ https://github.com/documaster/noark-extraction-validator/pull/66/commits/bad998fa5c1d4df46e42c6aa1d9f47d6d84ae284.

As part of the same commit I also extracted the InputStream to its own statement in the try-with-resources so that we free the resources properly